src/components/__tests__/FlawHistory.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { VueWrapper, mount } from '@vue/test-utils';
+
+import FlawHistory from '@/components/FlawHistory.vue';
+
+import type { ZodFlawHistoryItemType } from '@/types/zodFlaw';
+
+import { osimEmptyFlawTest, osimFullFlawTest } from './test-suite-helpers';
+
+function sampleHistoryItem(): ZodFlawHistoryItemType {
+  return {
+    pgh_created_at: '2024-10-04T12:06:56.760289Z',
+    pgh_slug: 'osidb.FlawAudit:123456',
+    pgh_label: 'update',
+    pgh_context: { url: 'osidb/api/v1/flaws/12d3820a-a43b-417c-a22e-46e47c232a63', user: 1 },
+    pgh_diff: { owner: ['', 'noemail@example.com'] },
+  };
+}
+
+describe('flawHistory', () => {
+  let subject: VueWrapper<InstanceType<typeof FlawHistory>>;
```

Comment:
since `subject` is not used outside of the tests, do you mind removing it from here and adding the `const` to the instantiation?